homeassistant/components/ubee/device_tracker.py

```diff
@@ -14,47 +14,48 @@
     CONF_HOST, CONF_PASSWORD, CONF_USERNAME)
 import homeassistant.helpers.config_validation as cv
 
-REQUIREMENTS = ['pyubee==0.2']
+REQUIREMENTS = ['pyubee==0.3']
 
 _LOGGER = logging.getLogger(__name__)
 
+CONF_MODEL = 'model'
+
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_HOST): cv.string,
     vol.Required(CONF_PASSWORD): cv.string,
     vol.Required(CONF_USERNAME): cv.string,
+    vol.Required(CONF_MODEL): cv.string,
```

Comment:
```suggestion
    vol.Optional(CONF_MODEL, default=DEFAULT_MODEL): cv.string,
```
You can avoid breaking change by given a default value for CONF_MODEL

Reply:
Thanks for suggestion.
That's what I did before I created this PR, but then I thought what criteria should I use to choose which model should be default. This component was released 16 days ago and supports 2 models only at the moment, but there are many models there and I thought it's better to make this change now while it's still new component.

Reply:
Not giving a default will also force users to look it up. If you supply a default and the user has a different model, s/he will most likely think it isn't working and either stop trying or create an issue it isn't working for him/her.